types/react-dom/index.d.ts

```diff
@@ -22,6 +22,8 @@ import {
     DOMAttributes, DOMElement, ReactNode, ReactPortal
 } from 'react';
 
+export { createRoot, hydrateRoot } from "./client";
```

Comment:
These warn when being imported from `react-dom`. I don't think we should add these here since they might be auto-imported now from `react-dom` instead of `react-dom/client`. But they should definitely be annotated as `@deprecated`